packages/mwap/lib/utils/get-environment-variables.js

```diff
@@ -0,0 +1,10 @@
+const dotenv = require("dotenv").config();
+
+function getEnvironmentVariables() {
+  return Object.entries(dotenv.parsed).reduce(
```

Comment:
Let's only add vars prefixed with `MWAP_`. This will help prevent accidental secrets meant for the server ending up in client bundles.

Reply:
Thinking of the best way to do this 🤔 Should we add separate DefinePlugin instances to both the server and client configs?

Reply:
Nope, only "DefinePlugin" vars that should be in the client bundle, i.e the `MWAP_` ones. The rest can be "dotenv"'d at runtime for the server. No reason to bundle secrets with the server as they can be accessed at runtime from the ".env" file.